proto/sources.proto

```diff
@@ -213,7 +210,6 @@ message S3 {
     credentials.KeySecret access_key = 1;
     credentials.Unauthenticated unauthenticated = 2;
     credentials.CloudEnvironment cloud_environment = 4;
-    credentials.AWSSessionTokenSecret session_token = 5;
```

Comment:
Why was this removed? The AWS test is failing now

Reply:
this must have been an accident, i don't recall touching these

Reply:
i think when i ran the generate protos command it deleted these, restoring now.